src/router.ts

```diff
@@ -1,6 +1,6 @@
 import VueRouter, { NavigationGuard } from "vue-router"
 
-import Landing from "./pages/LandingTilda.vue"
+import Landing from "./pages/Landing.vue"
```

Comment:
This change is not necessary.

```suggestion
import Landing from "./pages/LandingTilda.vue"
```